package.json

```diff
@@ -35,6 +35,10 @@
   "dependencies": {
     "async": "~0.9.0",
     "gapitoken": "~0.1.2",
+    "http-string-parser": "0.0.4",
+    "memorystream": "^0.3.1",
+    "multipart-stream": "^1.0.0",
```

Comment:
This module has a bug that fills the entire contents of the stream into memory, we previously used it and now have ripped it out of the library. I don't believe you'll need any of this stream functionality anyway because uploads are not supported by the batch API.